### ISSUE TITLE
Taking `Symbol` of LHS of transformed column triggering error

### ISSUE THREAD

**Issue by brayellison** [JuliaStats/MixedModels.jl#613]
I've got a formula where I'm applying `log` to the lhs and then calling `predict` with a new DataFrame that's had the random effects column updated. After doing so, it returns the error `ArgumentError: Response column must be initialized to a non-missing numeric value.`

[I believe it's caused by this line](https://github.com/JuliaStats/MixedModels.jl/blob/b106430df02466eb2fdfa2f8ff4bb858c086ee6e/src/predict.jl#L105) where `Symbol` is being applied. To reproduce, you can run the following:

```julia
f = @formula(log(lhs) ~ rhs)
Symbol(f.lhs)

# Symbol(":(log(lhs))")
```

It then checks if `respname` is in the columns and if it has any missing values in `newdata`. In my case the name `lhs` is in the columns but `log(lhs)` is not. I was able to get `predict` to work if I log transformed the variable first and then used that in the formula.

**Comment by palday** [comment]
Thanks for reporting this. There's a fix in #614 -- can you verify that it fixes the issue for you? You can install this version with:

```julia
julia>]
pkg> add MixedModels#pa/predsym
```

After it works and we've released a new version, you can go back to the released version with:

```julia
julia>]
pkg> free MixedModels
```

**Comment by brayellison** [comment]
Yup, looks like it worked. Thanks!
